lib/dpul_collections_web/router.ex

```diff
@@ -13,7 +13,7 @@ defmodule DpulCollectionsWeb.Router do
   scope "/", DpulCollectionsWeb do
     pipe_through :browser
 
-    get "/", PageController, :home
```

Comment:
you mentioned deleting the controller, etc, also. I would be in favor of that.

Reply:
Removed.